comixed-rest-api/src/main/java/org/comixedproject/controller/library/LibraryController.java

```diff
@@ -133,16 +133,19 @@ public void convertComics(@RequestBody() ConvertComicsRequest request) {
     ArchiveType archiveType = request.getArchiveType();
     boolean renamePages = request.isRenamePages();
     boolean deletePages = request.isDeletePages();
+    boolean deleteOriginalComic = request.isDeleteOriginalComic();
```

Comment:
See previous related comment: can we shorten this to deleteOriginal?

Reply:
ok